Resources/config/security.xml

```diff
@@ -9,8 +9,7 @@
         </service>
         <service id="sonata.media.security.superadmin_strategy" class="Sonata\MediaBundle\Security\RolesDownloadStrategy">
             <argument type="service" id="translator"/>
-            <argument/>
-            <!-- Either security.context or security.authorization_checker -->
+            <argument type="service" id="security.authorization_checker"/>
```

Comment:
if the empty argument is replaced in the compiler pass or the extension, we should remove that code, too

can you please check this @jenyak